spec/features/recently_viewed_spec.rb

```diff
@@ -1,19 +1,22 @@
 RSpec.feature 'Recently Viewed Products', :js do
   background do
     %w(Mug Shirt Jersey).each { |name| create(:product, name: name.to_s) }
+    create(:store, default: true)
   end
 
   scenario 'keep track of recently viewed products', js: true do
-    visit spree.root_path
-    click_link 'Mug'
-    wait_for_ajax
-    click_link 'Home'
-    click_link 'Jersey'
-    wait_for_ajax
-    click_link 'Home'
-    within(:css, 'ul#recently_viewed_products') do
-      expect(page).to have_text 'Mug'
-      expect(page).to have_text 'Jersey'
+    if Spree.version.to_f < 4.1
```

Comment:
Could you add scenario for Spree.version >= 4.1 ? :)

Reply:
with pleasure, but since 4.1 we don't have `#products[data-hook]` 😞 

here is override: https://github.com/spree-contrib/spree_recently_viewed/blob/master/app/overrides/spree/shared/_products/add_recently_viewed_products_to_products_index.rb#L4

and here view from `4.1` version: https://github.com/spree/spree/blob/v4.1.0/frontend/app/views/spree/shared/_products.html.erb

Reply:
👌 Thank you for the explanation :)